fixtures/varytec/giga-bar-hex-3.json

```diff
@@ -164,9 +172,190 @@
           "type": "NoFunction"
         },
         {
-          "dmxRange": [15, 255],
+          "dmxRange": [15, 22],
+          "type": "ColorPreset",
+          "comment": "Red",
+          "colors": ["#ff0000"]
+        },
+        {
+          "dmxRange": [23, 30],
+          "type": "ColorPreset",
+          "comment": "Green",
+          "colors": ["#00ff00"]
+        },
+        {
+          "dmxRange": [31, 40],
+          "type": "ColorPreset",
+          "comment": "Blue",
+          "colors": ["#0000ff"]
+        },
+        {
+          "dmxRange": [40, 47],
+          "type": "ColorPreset",
+          "comment": "Pure White",
+          "colors": ["#ffffff"]
+        },
+        {
+          "dmxRange": [48, 55],
+          "type": "ColorPreset",
+          "comment": "Amber",
+          "colors": ["#ffaa00"]
+        },
+        {
+          "dmxRange": [55, 62],
+          "type": "ColorPreset",
+          "comment": "UV",
+          "colors": ["UV"]
+        },
+        {
+          "dmxRange": [63, 70],
+          "type": "ColorPreset",
+          "comment": "Green",
+          "colors": ["#00ff00"]
+        },
+        {
+          "dmxRange": [71, 78],
+          "type": "ColorPreset",
+          "comment": "Fuchia",
+          "colors": ["#ff00ff"]
+        },
+        {
+          "dmxRange": [79, 86],
+          "type": "ColorPreset",
+          "comment": "Pink",
+          "colors": ["#ff00ff"]
```

Comment:
Please don't reuse the same hex color value for two differently named color presets in the same fixture. You can use https://colornames.org/search/ to find hex values for a given name.